src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,35 @@
+import { useContext } from 'react';
+import { TodosContext } from '../../contextes/TodosContext';
+import { Todo } from '../../types/Todo';
+import { TodoFilter } from '../TodoFilter/TodoFilter';
+
+export const Footer = () => {
+  const { todos, setTodos } = useContext(TodosContext);
+
+  const revomeCompletedTodos = () => setTodos(todos
+    .filter(todo => !todo.completed));
```

Comment:
would be better if you do it with functional `setState`
```suggestion
  const revomeCompletedTodos = () => setTodos((currentTodos) =>
    currentTodos.filter(todo => !todo.completed)
);
```

Reply:
do it in all cases